src/explorer/send_query.ts

```diff
@@ -0,0 +1,23 @@
+import { request } from 'graphql-request';
+import { buildTxnQuery } from './gql/query_builder.js';
```

Comment:
was getting errors when I just did `./gql/query_builder` or `./gql/query_builder.ts`. is this normal to need to have it as a js file even though the native file is ts?